### DISCUSSION TITLE
Owncast through Reverse Proxy

### DISCUSSION THREAD

**Discussion by Jerlag01** [owncast/owncast#535] [Q&A] [answered]
Hi,

I'm new to Owncast and I'm trying to run owncast through the reverse proxy of Nginx. 
Everything works fine except for the chat. It's disabled when calling up the website through the normal link and when going to the port directly it works fine. So my question is, Does anyone have an idea on how to enable the chat while using the normal link which is going through the reverse proxy?

**Comment by aaronpk** [comment, marked as answer]
Yep! See #430

**Reply by gabek** [reply]
The result of this was a little blurb that hopefully will point you in the right direction: https://owncast.online/docs/troubleshooting/#chat-is-disabled

**Comment by geekgonecrazy** [comment]
I would also say something I used today when setting up an owncast... caddy made this soooooooo easy.

```
caddy reverse-proxy --from live.mydomain.com --to 127.0.01:8080
```

it setup ssl and everything.

**Comment by remram44** [comment]
Apache 2.4:

```
ProxyPass /entry ws://127.0.0.1:8080/entry
ProxyPassReverse /entry ws://127.0.0.1:8080/entry
ProxyPass / http://127.0.0.1:8080/
ProxyPassReverse / http://127.0.0.1:8080/
ProxyPreserveHost On
ProxyRequests Off
```
